app/assets/javascripts/app/views/feedback_view.js

```diff
@@ -40,7 +43,40 @@ app.views.Feedback = app.views.Base.extend({
     if(evt) { evt.preventDefault(); }
     if(!window.confirm(Diaspora.I18n.t("reshares.post", {name: this.model.reshareAuthor().name}))) { return }
     this.model.interactions.reshare();
-  }
+  },
+
+  blockUser: function(evt) {
+    if(evt) { evt.preventDefault(); }
+    if(!confirm(Diaspora.I18n.t('ignore_user'))) { return; }
+
+    this.model.blockAuthor()
+      .done(function() {
+        // return to stream
+        document.location.href = "/stream";
```

Comment:
On second thought, I guess this might be solved by backbonification of the stream.